Sources/Datadog/Core/System/MobileDevice.swift

```diff
@@ -100,14 +104,24 @@ internal class MobileDevice {
     }
 }
 
+/// Observes "Low Power Mode" setting changes and provides `isLowPowerModeEnabled` value in a thread-safe manner.
+///
+/// Note: this was added in https://github.com/DataDog/dd-sdk-ios/issues/609 where `ProcessInfo.isLowPowerModeEnabled` was considered
+/// not thread-safe on iOS 15. With this monitor, we change from pulling to push model for reading this property. Now, it will never be read simultaneously
+/// by multiple SDK threads - instead it will be read only once after LMP setting change and bridged to other threads through thread-safe `ValuePublisher`.
```

Comment:
```suggestion
/// by multiple SDK threads - instead it will be read only once after LPM setting change and bridged to other threads through thread-safe `ValuePublisher`.
```